lib/modules/apostrophe-images/lib/cursor.js

```diff
@@ -40,6 +40,10 @@ module.exports = {
       }
     });
 
+    // Filter. Get images depending of the
```

Comment:
Typo: "depending on the"

Reply:
We aren't telling them the options to use in this documentation.

Reply:
@abea I'm not sure to understand what options you are talking about ? The user has nothing to configure since these filters have been added in `apostrophe-images` directly.

Reply:
Can they not use the filters in API calls?

Reply:
You you're right, I no longer had that in mind. 
Just improved comments.